.github/workflows/ci.yaml

```diff
@@ -24,14 +22,21 @@ jobs:
           - convenient_test_dev
           - convenient_test_manager
           - convenient_test_manager_dart
+        version: ["3.0.0", ""]
+        channel: [stable, beta, ""]
+        exclude:
+          - { version: "3.0.0", channel: "stable" }
+          - { version: "3.0.0", channel: "beta" }
+          - { version: "", channel: "" }
```

Comment:
What about this:

```yaml
channel_version:
- {channel: stable}
- {channel: beta}
- {version: 3.0.0}
```

Instead of matrix + exclude

Reply:
I did so in [this commit](https://github.com/fzyzcjy/flutter_convenient_test/pull/274/commits/3d6027f159d69804119f207db993e802091598e1), but unfortunately it didn't work as intended. Or maybe I did something wrong?

Reply:
Oh, nice.

I spent way too much time trying to fix the Action but my solution was overcomplicated anyways. Yours is simpler and works just as good. Thanks:)